neo/rawio/intanrawio.py

```diff
@@ -201,6 +205,7 @@ def _parse_header(self):
 
         # signals
         signal_channels = []
+        self.native_channel_order = dict()
```

Comment:
```suggestion
        self.native_channel_order = {}
```

I know others like `dict()`, but `{}` is faster. [review here](https://madebyme.today/blog/python-dict-vs-curly-brackets/)

I like {} better, but since the performance is only about 2x faster if you really prefer dict() we can keep it.

Reply:
I will change this on the next commit.

Reply:
Done.